doc/CONTRIBUTING.md

```diff
@@ -102,6 +102,20 @@ Good commit message subject :
 Document how to install the project
 ```
 
+Also, when you specify what you did avoid commit message subjects with "Fix bug
+in such and such feature". Saying you are fixing something implies the previous
+implementation was wrong and yours is right, which might not be even true.
+Instead, humbly say what you did technically and **then** explain how this is
```

Comment:
I definitely totally agree on what you say.
Im' just saying that a phrase like the ones you just did:
> the shortlog is what your doing, and thus stating unquestionable facts

...explains things _way_ better than the previous one (with the part about «being humble»), which introduce a rather emotional language (that could be mistaken as defensive).